### DISCUSSION TITLE
how to get the default parameter set?

### DISCUSSION THREAD

**Discussion by aiinibxl** [iiasa/CWatM#84] [Q&A] [answered]
Hello, everyone~
In CWatM, 12 parameters need to be calibrated, but calibration is time-consuming if I want to conduct a global simulation. So, I would like to ask how can I get the default values of these parameters for a global run?

![1691655579735](https://github.com/iiasa/CWatM/assets/96671336/b22fe061-8f3a-4fe4-abab-d6711346bff1)

**Comment by mikhailsmilovic** [comment, marked as answer]
Guten Tag, **these values** in this settings file (settings_CWatM_template_30min.ini) from the repository https://github.com/iiasa/CWatM-Earth-30min **are the default values for a global run** at 30 arc minutes. Let us know how it goes!
